SECP-service/src/main/resources/assets/app/js/portal/configure-controller.js

```diff
@@ -0,0 +1,76 @@
+angular.module('SECP')
+    .controller('ConfigureController', ['$scope', function ($scope) {
+        var permissiontable =$('#permissionTable').DataTable();
```

Comment:
For the future, look up let, const and their differences between var. Basically var lets you redeclare the variable, while let wont. This encourages that variables only exist when they should.